cmd/yurt-static-pod-upgrade/static-pod-upgrade.go

```diff
@@ -0,0 +1,91 @@
+/*
+Copyright 2023 The OpenYurt Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"math/rand"
+	"os"
+	"time"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"k8s.io/klog/v2"
+
+	"github.com/openyurtio/openyurt/pkg/projectinfo"
+	upgrade "github.com/openyurtio/openyurt/pkg/static-pod-upgrade"
+)
+
+var (
+	manifest string
+	mode     string
+)
+
+func main() {
+	rand.Seed(time.Now().UnixNano())
+	version := fmt.Sprintf("%#v", projectinfo.Get())
+	cmd := &cobra.Command{
+		Use: "yurt-static-pod-upgrade",
```

Comment:
I think we don't need to add a new command, how about add a sub command for upgrading static pod in node-servant component?

Reply:
Agree! I will move it to node-servant.

Reply:
Done